package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "askar-webcrypto",
+  "name": "@animo-id/askar-webcrypto",
   "version": "0.0.0",
   "main": "build/index.js",
-  "author": "Animo Solutions <development@animo.id> (https://animo.id)",
+  "author": "Animo Solutions <development@animo.id> (https://www.animo.id)",
```

Comment:
Woot. https://www.animo.id doesn't seem to work. I'll look into into it, but why not just https://animo.id?

Reply:
lol thats weird, we are not on the world wide web!!!!

https://animo.id works as well, I prefer the looks of `www` haha.